cmd/swarm/config.go

```diff
@@ -74,8 +74,6 @@ const (
 	SWARM_ENV_ENS_API                 = "SWARM_ENS_API"
 	SWARM_ENV_ENS_ADDR                = "SWARM_ENS_ADDR"
 	SWARM_ENV_CORS                    = "SWARM_CORS"
-	SWARM_ENV_BOOTNODES               = "SWARM_BOOTNODES"
```

Comment:
I am not sure why we have environment variables at all, since we have a config file and command line arguments.

Why do we need a third way to override default parameters, and what is the precedence? cc @holisticode

Reply:
Any feedback on this? If not, I will go ahead and remove these as we are just polluting the code with unused code and assignments.

Reply:
@nonsense check the discussion here: https://gitter.im/ethersphere/orange-lounge?at=5a0c73a7df09362e67110c89

it was at some point formulated as a requirement that our configuration would allow :"....typically, command line flags take precedence, then env vars, then config file, all for the same set of parameters"

Reply:
So I agree with @zelig here, if it's there, and we continue supporting env vars, then we should check this specific var, and it should not be deleted

Reply:
My personal opinion is that we don't need these env variables at all, and are just complicating the code. Does anyone have any idea if any of this is actually being used and why we implemented it in the first place?

For now, I will revert this particular change and I have created an issue to address this.

Fixing this is out of scope for this PR.